.github/workflows/build_wheels_linux.yml

```diff
@@ -265,22 +275,42 @@ jobs:
           export PYTORCH_VERSION="$(${CONDA_RUN} pip show torch | grep ^Version: | sed 's/Version: *//' | sed 's/+.\+//')"
           ${CONDA_RUN} python setup.py clean
           echo "Successfully ran `python setup.py clean`"
-          if [[ "$BUILD_VERSION" != *"+"${CU_VERSION} ]]; then
-            BUILD_VERSION="${BUILD_VERSION}+${CU_VERSION}"
+          if [[ ${{ inputs.is-release-wheel }} == true || ${{ inputs.is-release-tarball }} == true ]]; then
```

Comment:
Think its better to have the tarballs in a separate workflow. I think they can run on each build similar to wheel if we want but no need to force it in here